backend/json/603050002.json

```diff
@@ -3,22 +3,49 @@
 	"name": "Urban Sanitation",
 	"version": "1.0.0",
 	"cascades": [
-		"/sqlite/cascade-296940912-v2.sqlite",
-		"/sqlite/kenya_organizations.sqlite"
```

Comment:
why you remove this?

Reply:
sorry, I think I didn't merge the main before I did translations on this form

Reply:
I've fixed it on this commit
https://github.com/akvo/nmis-mobile/pull/61/commits/0be5faa364ec933547a13c220bfb10cccf2cf4be